src/test/java/com/commercetools/sync/products/utils/productvariantupdateactionutils/prices/PriceFixtures.java

```diff
@@ -0,0 +1,211 @@
+        null, byMonth(1), byMonth(2), null, null);
+
+    public static final Price UK_111_GBP_02_03 = getPrice(BigDecimal.valueOf(111), GBP, UK,
+        null, byMonth(2), byMonth(3), null, null);
+
+    public static final Price UK_333_GBP_02_05 = getPrice(BigDecimal.valueOf(333), GBP, UK,
+        null, byMonth(2), byMonth(5), null, null);
+
+    public static final Price US_555_USD_CUST2_01_02 = getPrice(BigDecimal.valueOf(555), USD, US,
+        "cust2", byMonth(1), byMonth(2), null, null);
+
+    public static final Price USD_555_CUST2_CHANNEL1_01_02 = getPrice(BigDecimal.valueOf(555), USD, null,
+        "cust2", byMonth(1), byMonth(2), "channel1", null);
+
+    public static final Price US_555_USD_CUST2_CHANNEL1_01_02 = getPrice(BigDecimal.valueOf(555), USD, US,
+        "cust2", byMonth(1), byMonth(2), "channel1", null);
+
+    public static final Price FR_777_EUR_01_04 = getPrice(BigDecimal.valueOf(777), EUR, FR,
+        null, byMonth(1), byMonth(4), null, null);
+
+    public static final Price NE_123_EUR_01_04 = getPrice(BigDecimal.valueOf(123), EUR, NE,
+        null, byMonth(1), byMonth(4), null, null);
+
+    public static final Price NE_321_EUR_04_06 = getPrice(BigDecimal.valueOf(321), EUR, NE,
+        null, byMonth(4), byMonth(6), null, null);
+
+    public static final Price DE_666_EUR = getPrice(BigDecimal.valueOf(666), EUR, DE, null, null, null, null, null);
+
+    public static final Price DE_222_EUR_01_02_CHANNEL1_CUSTOMTYPE1_CUSTOMFIELDY = getPrice(BigDecimal.valueOf(222),
+        EUR, DE, null, byMonth(1), byMonth(2),
+        "channel1", getMockCustomFields("customType1", "foo", JsonNodeFactory.instance.textNode("Y")));
+
+    public static final Price DE_222_EUR_01_02_CHANNEL2_CUSTOMTYPE2_CUSTOMFIELDX = getPrice(BigDecimal.valueOf(222),
+        EUR,
+        DE, null,
+        byMonth(1), byMonth(2),
+        "channel2", getMockCustomFields("customType2", "foo", JsonNodeFactory.instance.textNode("X")));
+
+    public static final Price UK_22_GBP_CUSTOMTYPE1_CUSTOMFIELDY = getPrice(BigDecimal.valueOf(22), GBP,
+        UK, null, null, null, null,
+        getMockCustomFields("customType1", "foo", JsonNodeFactory.instance.textNode("Y")));
+
+    public static final Price UK_22_USD_CUSTOMTYPE2_CUSTOMFIELDX = getPrice(BigDecimal.valueOf(22), USD,
+        UK, null, null, null, null,
+        getMockCustomFields("customType2", "foo", JsonNodeFactory.instance.textNode("X")));
+
+    public static final Price UK_1_GBP_CHANNEL1_CUSTOMTYPE1_CUSTOMFIELDX = getPrice(BigDecimal.valueOf(1), GBP,
+        UK, null, null, null, "channel1",
+        getMockCustomFields("customType1", "foo", JsonNodeFactory.instance.textNode("X")));
+
+    public static final Price DE_22_USD = getPrice(BigDecimal.valueOf(22), USD, DE,
+        null, null, null, null, null);
+
+
+    @Nonnull
+    private static Price getPrice(@Nonnull final BigDecimal value,
+                                  @Nonnull final CurrencyUnit currencyUnits,
+
```

Comment:
Why blank line here?

Reply:
that was weird. Thanks for noticing: 82af316